front/src/applications/operationalStudies/views/Scenario/components/ImportTimetableItem/generateTrainSchedulesPayloads.ts

```diff
@@ -18,14 +18,16 @@ function generateTrainSchedulePayload(train: GraouTrainSchedule): TrainSchedule
       if (validUICNumber) {
         acc.path.push({
           id: stepId,
-          uic: Number(step.uic),
-          secondary_code: step.chCode,
+          location: {
+            operational_point: { uic: Number(step.uic), secondary_code: step.chCode ?? '' },
+          },
         });
       } else {
         acc.path.push({
           id: stepId,
-          trigram: step.name, // we use ocpRef when uic is NaN
-          secondary_code: step.chCode ?? '',
+          location: {
+            operational_point: { trigram: step.trigram ?? '', secondary_code: step.chCode ?? '' },
```

Comment:
We were using `step.name` as the trigram before, now we're using `step.trigram`. I'm worried about changing behavior here: is the new behavior identical to the old one? If not, it would be safer to do the same as before (and consider changing it in a separate commit if it makes sense to, so that it can be tested/reverted independently).